Networking/Networking/Remote/GenerativeContentRemote.swift

```diff
@@ -199,80 +200,71 @@ private extension GenerativeContentRemote {
-                slug: "",
-                permalink: "",
-                date: Date(),
-                dateCreated: Date(),
-                dateModified: nil,
-                dateOnSaleStart: nil,
-                dateOnSaleEnd: nil,
-                productTypeKey: ProductType.simple.rawValue,
-                statusKey: ProductStatus.published.rawValue,
-                featured: false,
-                catalogVisibilityKey: ProductCatalogVisibility.visible.rawValue,
-                fullDescription: "",
-                shortDescription: "",
-                sku: "",
-                price: "",
-                regularPrice: "",
-                salePrice: "",
-                onSale: false,
-                purchasable: false,
-                totalSales: 0,
-                virtual: false,
-                downloadable: false,
-                downloads: [],
-                downloadLimit: -1,
-                downloadExpiry: -1,
-                buttonText: "",
-                externalURL: "",
-                taxStatusKey: ProductTaxStatus.taxable.rawValue,
-                taxClass: "",
-                manageStock: false,
-                stockQuantity: nil,
-                stockStatusKey: ProductStockStatus.inStock.rawValue,
-                backordersKey: ProductBackordersSetting.notAllowed.rawValue,
-                backordersAllowed: false,
-                backordered: false,
-                soldIndividually: false,
-                weight: "",
-                dimensions: ProductDimensions(length: "", width: "", height: ""),
-                shippingRequired: true,
-                shippingTaxable: true,
-                shippingClass: "",
-                shippingClassID: 0,
-                productShippingClass: nil,
-                reviewsAllowed: true,
-                averageRating: "",
-                ratingCount: 0,
-                relatedIDs: [],
-                upsellIDs: [],
-                crossSellIDs: [],
-                parentID: 0,
-                purchaseNote: "",
-                categories: [],
-                tags: [],
-                images: [],
-                attributes: [],
-                defaultAttributes: [],
-                variations: [],
-                groupedProducts: [],
-                menuOrder: 0,
-                addOns: [],
-                isSampleItem: false,
-                bundleStockStatus: nil,
-                bundleStockQuantity: nil,
-                bundledItems: [],
-                compositeComponents: [],
-                subscription: nil,
-                minAllowedQuantity: nil,
-                maxAllowedQuantity: nil,
-                groupOfQuantity: nil,
-                combineVariationQuantities: nil)
+
+        let tagsAsString = {
+            guard !tags.isEmpty else {
+                return ""
+            }
+
+            return
+                ", tags:Given the list of available tags ```\(tags.map { $0.name }.joined(separator: ", "))```, " +
```

Comment:
❓ Any reason why we don't have a space after the colon? Is that to make AI work?